### PR TITLE
A language category for OPFOR loadout

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request
Tin. Adds language unavailable from the round-start selection to a OPFOR loadout category.

## How This Contributes To The Skyrat Roleplay Experience
May someone desire these for their OPFOR plot, its easier to be given like this rather than asked for separately through admin chat.

## Proof of Testing

rUdBky54pj

## Changelog

:cl:
add: Adds a language category to the OPFOR loadout selection
/:cl:

### CODE REVIEW

**Comment by lessthnthree** [comment]
This should be a CL add, not a code.

File: modular_skyrat/modules/opposing_force/code/equipment/languages.dm [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/datum/opposing_force_equipment/language
+	category = OPFOR_EQUIPMENT_CATEGORY_LANGUAGES
+	var/datum/language/language
```
Comment (Zonespace27) [inline modular_skyrat/modules/opposing_force/code/equipment/languages.dm]: code doc, also this isn't an instance, just a type

File: modular_skyrat/modules/opposing_force/code/equipment/languages.dm [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+/datum/opposing_force_equipment/language/on_issue(mob/living/target)
+	target.grant_language(language, TRUE, TRUE, LANGUAGE_MIND)
+
+/datum/opposing_force_equipment/language/codespeak
+	name = "Codespeak"
+	description = "Syndicate operatives can use a series of codewords to convey complex information, while sounding like random concepts and drinks to anyone listening in."
+	item_type = /obj/effect/gibspawner/generic
```
Comment (Zonespace27) [inline modular_skyrat/modules/opposing_force/code/equipment/languages.dm]: move this to parent

File: modular_skyrat/modules/opposing_force/code/equipment/languages.dm [hunk 3]
```diff
@@ -1,16 +1,13 @@
 /datum/opposing_force_equipment/language
+	name = "Codespeak"
```
Comment (Zonespace27) [inline modular_skyrat/modules/opposing_force/code/equipment/languages.dm]: change this back, i meant SPECIFICALLY what is now line 5, not anything else

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**